src/test/java/org/stellar/sdk/requests/OffersRequestBuilderTest.java

```diff
@@ -19,4 +54,46 @@ public void testForAccount() {
             .buildUri();
     assertEquals("https://horizon-testnet.stellar.org/accounts/GBRPYHIL2CI3FNQ4BXLFMNDLFJUNPU2HY3ZMFSHONUCEOASW7QC7OX2H/offers?limit=200&order=desc", uri.toString());
   }
+
+  @Test
+  public void testWithoutParams() {
+    Server server = new Server("https://horizon-testnet.stellar.org");
+    HttpUrl uri = server.offers().buildUri();
+    assertEquals("https://horizon-testnet.stellar.org/offers", uri.toString());
+  }
+
+  @Test
+  public void testForSeller() {
+    Server server = new Server("https://horizon-testnet.stellar.org");
+    HttpUrl uri = server.offers().forSeller("GBRPYHIL2CI3FNQ4BXLFMNDLFJUNPU2HY3ZMFSHONUCEOASW7QC7OX2H").buildUri();
+    assertEquals("https://horizon-testnet.stellar.org/offers?seller=GBRPYHIL2CI3FNQ4BXLFMNDLFJUNPU2HY3ZMFSHONUCEOASW7QC7OX2H", uri.toString());
+  }
+
+  @Test
+  public void testForSellingAsset() {
```

Comment:
could you add a test case for filtering by the native selling asset and another test case for filtering by the native buying asset?

Reply:
also, can you add a test case for filtering by BOTH buying and selling assets since that is supported by the new endpoint?